### ISSUE TITLE
False positive in Readability.ParenthesesInCondition

### ISSUE THREAD

**Issue by kerryb** [rrrene/credo#515]
### Environment

* Credo version (`mix credo -v`): 0.8.10
* Erlang/Elixir version (`elixir -v`):
```
Erlang/OTP 20 [erts-9.2.1] [source] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]

Elixir 1.6.3 (compiled with OTP 19)
```
* Operating system: macOS 10.13.3

### What were you trying to do?

Execute checks against the following code (reduced test case):

```elixir
defmodule Foo do
  @moduledoc """
  Reduced test case
  """

  def bar(a, b) do
    # The next line is the one the error is incorrectly reported against
    if (a + b) / 100 > threshold(), do: :high, else: :low
  end

  def threshold, do: 50
end
```

I think the key is the fact that the if condition begins with an open paren (for operator precedence), and ends with a close paren (on a function call). Credo seems to make the incorrect assumption that these form a pair surrounding the condition.

### Expected outcome

No errors

### Actual outcome

    ┃ [R] ↗ The condition of `if` should not be wrapped in parentheses.
    ┃       lib/foo.ex:8:5 #(Foo.bar)

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It seems to be already fixed on `master` and in `0.9.0-rc8`.

You can try `master` by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍

**Comment by kerryb** [comment]
Thanks for the quick reply, but I still see the error when I switch to master (mix picked up 261862b).

**Comment by rrrene** [comment]
I see. It should now be fixed. Sorry for the delay! 👍
